app/src/main/java/org/fossasia/phimpme/editor/view/RotateImageView.java

```diff
@@ -75,29 +75,31 @@ public void draw(Canvas canvas) {
 		super.draw(canvas);
 		if (bitmap == null)
 			return;
-		maxRect.set(0, 0, getWidth(), getHeight());// The maximum bounding rectangle
-
+		maxRect.set(0, 0, (int)originImageRect.width(), (int)originImageRect.height());// The maximum bounding rectangle
 		calculateWrapBox();
 		scale = 1;
 		if (wrapRect.width() > getWidth()) {
-			scale = getWidth() / wrapRect.width();
-		}
+            scale=getWidth()/wrapRect.width();
```

Comment:
Indent the codes properly.

Reply:
@anantprsd5 Yes, I'll do that. I wanted to know if the implementation for rotation is fine now or should be changed further?